ivy.el

```diff
@@ -3831,7 +3829,7 @@ Don't finish completion."
   (interactive)
   (delete-minibuffer-contents)
   (if (and ivy--directory
-           (string-match "/$" (ivy-state-current ivy-last)))
+           (string-match-p "/$" (ivy-state-current ivy-last)))
```

Comment:
Should the regexp here be:
```el
"/\\'"
```
?

Reply:
As with replacement of `^`, there is no difference if we're matching against a single line. I don't have a strong opinion on the matter. Do you feel it should be addressed?